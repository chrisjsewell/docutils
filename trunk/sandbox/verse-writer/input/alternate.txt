| alternative
|   intends
| indent
|   altering
| since
