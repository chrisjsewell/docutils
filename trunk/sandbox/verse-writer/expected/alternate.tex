\begin{verse}
alternative \\
\vin intends \\
indent \\
\vin altering \\
since \\
\end{verse}
